src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -429,13 +441,19 @@ export const Checkout = ({
         setSignUiState('MANUAL_REVIEW')
         return
       }
-
+      trackOfferEvent({
+        eventName: EventName.SignError,
+      })
```

Comment:
Sorry for dragging on but I just wonder why we don't include error type here? Like

```
trackOfferEvent({
  eventName: EventName.SignError,
  options: { error, errorType: ErrorEventType.ManualReviewRequired },
})
```

Seems like if we move it inside the if-statement that would work, no?